AlphaWallet/TokenScriptClient/Models/AssetAttributeSyntaxValue.swift

```diff
@@ -258,6 +262,50 @@ extension Dictionary where Key == AttributeId, Value == AssetAttributeSyntaxValu
         self["traits"]?.value
     }
 
+    var meltValue: String? {
+        self["meltValue"]?.stringValue
+    }
+
+    var meltFeeRatio: BigInt? {
+        self["meltFeeRatio"]?.intValue
+    }
+
+    var meltFeeMaxRatio: BigInt? {
+        self["meltFeeMaxRatio"]?.intValue
+    }
+
+    var totalSupply: String? {
```

Comment:
Can you call it `totalSupplyStringValue` instead, if it returns `String?`? Just guarding against name clashes and also when we need to return different types for the same attribute.